host/platform/Windows/include/windows/force_compose.h

```diff
@@ -0,0 +1,21 @@
+/*
+Looking Glass - KVM FrameRelay (KVMFR) Client
+Copyright (C) 2017-2021 Geoffrey McRae <geoff@hostfission.com>
```

Comment:
I am not the author here :)
Please be sure to set correct copyrights where you make alterations/additions, etc.

Reply:
Fixed.